### DISCUSSION TITLE
Concatenating strings with numbers

### DISCUSSION THREAD

**Discussion by Ariemeth** [postmanlabs/postman-flows#354] [Q&A] [answered]
For an initial exploration into flows I have been trying to build a flow to create N test users.  Couldn't figure out how to do a few things.
* ~~Concatenate 2 strings in a field.  example incoming data "user1" -> "user1@example.com"~~
* Concatenate a string and a number.  example "user" + 1 => "user1"
 
I'm finding working around requests to be good, but generating user info for testing is painful right now as it is manual.

**Comment by samay-v** [comment, marked as answer]
Hey, we're working on a way to transpose datatype. But today you can't operate a string to a number

**Reply by GaetanVandenBergh** [reply]
Do you already know if the transposition will allow us to solve this #337 in a sense ?

**Reply by Ariemeth** [reply]
Good to know it is in work.
Thanks for letting me know it isn't possible yet.
